diff -Naur acidrip-0.14/AcidRip/acidrip.pm acidrip-0.14.tpg/AcidRip/acidrip.pm
--- acidrip-0.14/AcidRip/acidrip.pm	2004-07-25 14:03:09.000000000 +0000
+++ acidrip-0.14.tpg/AcidRip/acidrip.pm	2007-07-15 14:38:33.000000000 +0000
@@ -914,7 +914,7 @@
   'video_codec'             => 'lavc',
   'lavc_options'            => 'vcodec=mpeg4:vhq:v4mv:vqmin=2',
   'divx4_options'           => '',
-  'xvid_options'            => '',
+  'xvid_options'            => 'chroma_opt:vhq=4:bvhq=1:quant_type=mpeg',
   'vuv_options'             => '',
   'video_options'           => '',
   'video_bitrate'           => 0,
